### ISSUE TITLE
Epoch 500 and LJSpeech is not sure of "r" sound- cause for concern?

### ISSUE THREAD

**Issue by rachel-beeson-connex** [rhasspy/piper#171]
Hi all,

I've been using Piper to train a few voices and I had good success with a UK voice using about 20 hours of data at 16 kHz.

For my next experiment I wanted to build a model based on LJSpeech, sampled at 16kHz. Of course I had to change the phone set to espeak en-US to match the LJ accent.

My other voices converged around 800k steps which is where I'm at now for LJSpeech, but currently my model seems to have learned decent phones for everything except the word final "r" sounds. It's totally unpredictable whether they'll be said or not. At first I thought it might have something to do with using single r-colored vowel symbols instead of a separate vowel and r phone symbol, but I can't really see why this would be the case, if everything is consistent in the training data (and it is).

Is this normal? Will the voice continue to learn the r-sounds if I let it continue to train? I just worry because I didn't have this issue with the UK voices (maybe because they are non-rhotic, so I didn't have the chance to observe this phenomenon). 

Thanks.

**Comment by rachel-beeson-connex** [comment]
Nevermind, I am so very silly and was phonemizing my test set using en-GB espeak. :)

**Comment by synesthesiam** [comment]
Glad you got it resolved! I've had this exact problem: espeak phonemizes "r" in very different ways for U.S. and British English, it seems.
